caddy/Caddyfile

```diff
@@ -1,12 +1,12 @@
 # Listen on all interfaces, default Caddy port
-:2015 {
+{$SITE_ADDRESS} {
```

Comment:
Looks like this needs to be specified by the user. Able to document it?

Reply:
Added some documentation